BedrockPlugin.cpp

```diff
@@ -2,7 +2,7 @@
 #include "BedrockPlugin.h"
 
 // Global static values
-list<BedrockPlugin*>* BedrockPlugin::g_registeredPluginList = 0;
+list<BedrockPlugin*>* BedrockPlugin::g_registeredPluginList = nullptr;
```

Comment:
While we're wainitg for Tyler:
How do your old school see to add `const` to definitely const methods like [this](https://github.com/Expensify/Bedrock/blob/master/libstuff/libstuff.h#L328)?
And make return values `size_t` for methods returning size values, like [this](https://github.com/Expensify/Bedrock/blob/master/libstuff/libstuff.h#L139)?